package.json

```diff
@@ -32,6 +32,7 @@
     "tape": "^4.5.1"
   },
   "dependencies": {
+    "@google/cloud-diagnostics-common": "0.2.3",
     "google-auth-library": "0.9.8",
     "lodash": "4.13.1"
```

Comment:
@ofrobots There has been api breakages with lodash before so I always found pinning on a stable API to be the best, google auth library has been removed as a direct dependency now since we are using the shared diag code